resources/views/checkout/partials/sections/payment/payment-methods.blade.php

```diff
@@ -1,15 +1,21 @@
-<x-rapidez-ct::input.radio
-    name="payment_method"
-    v-bind:value="method.code"
-    v-bind:dusk="'method-'+index"
-    v-model="checkout.payment_method"
-    required
->
-    <div>@{{ method.title }}</div>
-    <img
-        class="max-h-10 w-12"
-        v-bind:alt="method.code"
-        v-bind:src="`/payment-icons/${method.code}.svg`"
-        onerror="this.onerror=null; this.src='/payment-icons/default.svg'"
-    />
-</x-rapidez-ct::input.radio>
\ No newline at end of file
+<template v-if="false"></template>
+@stack('payment_methods')
```

Comment:
This is so payment providers can add their own template here. e.g. https://github.com/rapidez/riverty/pull/1/files#diff-9551ccb8b1b6a35c1e8d9615c9245a6937fc9b1cea0837b54e30913cb026e1eaR19